client/interfaces.go

```diff
@@ -68,6 +68,9 @@ type ContainerServer interface {
 	MigrateContainer(name string, container api.ContainerPost) (op *Operation, err error)
 	DeleteContainer(name string) (op *Operation, err error)
 
+	ConsoleContainer(containerName string, console api.ContainerConsolePost, args *ContainerConsoleArgs) (*Operation, error)
+	GetContainerConsoleLog(containerName string, args *ContainerConsoleLogArgs) (io.ReadCloser, error)
+	DeleteContainerConsoleLog(containerName string, args *ContainerConsoleLogArgs) error
```

Comment:
All return values should be named in interfaces.go (for documentation).
I'll send a branch which fixes the few other missing ones we have now.

Reply:
done